common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -4,7 +4,8 @@
 @import templates.inlineJS.blocking.js._
 
 <!--[if lt IE 9]>
-    <script src="@Static("javascripts/components/html5shiv/html5shiv.js")"></script>
+    <script src="@Static("javascripts/ie8polyfills.js")"></script>
+    <script>@Html(polyfills().body)</script>
```

Comment:
don't think the polyfills can be scoped to this comment e.g. `raf` is needed by IE9, opera mini, android < 4.4 etc. but good to group them up here.